### PR TITLE
chore: user the driver from cdn

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: build_package.py [hunk 1]
```diff
@@ -15,12 +15,27 @@
...
+import stat
 import subprocess
 import sys
 import zipfile
 
 from playwright.path_utils import get_file_dirname
 
+driver_version = "0.5.2"
+
+if not os.path.exists("driver"):
```
Comment (mxschmitt) [inline build_package.py:27]: Duplicate logic with the paths. I would use Pathlib for that.

File: build_package.py [hunk 2]
```diff
@@ -15,12 +15,27 @@
...
+    os.makedirs("driver")
+if not os.path.exists("playwright/driver"):
+    os.makedirs("playwright/driver")
+
+for platform in ["mac", "linux", "win32", "win32_x64"]:
+    zip_file = f"playwright-cli-{driver_version}-{platform}.zip"
+    if not os.path.exists("driver/" + zip_file):
+        url = "https://playwright.azureedge.net/builds/cli/" + zip_file
+        print("Fetching ", url)
+        subprocess.check_call(["curl", url, "-o", "driver/" + zip_file])
```
Comment (mxschmitt) [inline build_package.py:37]: Let's do it in memory like here? https://github.com/microsoft/playwright-python/pull/172/files#diff-56e4178dc64fcedaf003454c830bc80cR61

File: build_package.py [hunk 3]
```diff
@@ -37,24 +52,46 @@
...
+    if platform == "win32_x64":
+        wheel = "win_amd64.whl"
     wheel_location = without_platform + wheel
     shutil.copy(base_wheel_location, wheel_location)
-    from_location = f"driver/out/{driver}"
-    to_location = f"playwright/drivers/{driver}"
-    with zipfile.ZipFile(wheel_location, "a") as zipf:
-        zipf.write(from_location, to_location)
+    with zipfile.ZipFile(wheel_location, "a") as zip:
+        for file in os.listdir(f"driver/{platform}"):
```
Comment (mxschmitt) [inline build_package.py:88]: Got it. So driver will get still distributed over wheel. Then curl is fine, maybe we should download it on-demand which seems better (source code install would work).

File: build_package.py (hunk 1 above)
Comment (pavelfeldman) [reply build_package.py:27]: Those are different folders or what do you mean?

File: build_package.py (hunk 3 above)
Comment (pavelfeldman) [reply build_package.py:88]: Yes, we still bundle.

**Pull request merged**